docs/development.md

```diff
@@ -188,6 +189,13 @@ KubeFed controller manager:
 
 Follow the [cleanup instructions in the user guide](../charts/kubefed/README.md#uninstalling-the-chart).
 
+## Embedding static files using go-bindata
+This project is using `go-bindata` tool for embedding static files into its e2e test-suite to enable the creation of a self-contained e2e binary.
+
+You can install this utility using the [download-binaries.sh](../scripts/download-binaries.sh#L70) script.
```

Comment:
Please remove the line number from the link so we don't have to maintain it. It should be sufficient to link to the script.